### PR TITLE
Update redis and syncd with notify logic

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

File: lib/src/sai_redis_switch.cpp [hunk 1]
```diff
@@ -1,19 +1,12 @@
 #include "sai_redis.h"
+#include "sairedis.h"
```
Comment (stcheng) [inline lib/src/sai_redis_switch.cpp]: two files with one named sai_redis.h one sairedis.h.
this might be confusing.

Comment (kcudnik) [inline lib/src/sai_redis_switch.cpp]: yes, i will change that later, new one will be distributes in the package, previous one is internal

File: player/player.cpp [hunk 2]
```diff
@@ -1084,14 +1098,27 @@ int replay(int argc, char **argv)
...
 {
-    std::cout << "Usage: player [-h] recordfile" << std::endl;
+    std::cout << "Usage: player [-h] recordfile" << std::endl << std::endl;
+    std::cout << "    -C --skipNotifySyncd:" << std::endl;
+    std::cout << "        Will not send notify init/apply view to syncd" << std::endl << std::endl;
+    std::cout << "    -d --enableDebug:" << std::endl;
+    std::cout << "        Enable syslog debug messages" << std::endl << std::endl;
     std::cout << "    -h --help:" << std::endl;
-    std::cout << "        Print out this message" << std::endl;
+    std::cout << "        Print out this message" << std::endl << std::endl;
```
Comment (kcudnik) [inline player/player.cpp]: yes for better readability

File: lib/src/sai_redis_switch.cpp (hunk 1 above)
Comment (stcheng) [inline lib/src/sai_redis_switch.cpp]: that's fine. i'll merge it. let's change that later soon.

**Pull request merged**